run.py

```diff
@@ -29,27 +29,57 @@ def run():
     filename = f"{video_info['title']}.mp3"
     output_path = os.path.join(where_to_save, filename)
 
-    options = {
-        # reduces the several line of output to barest minimal and therefore, time
-        'quiet': True,
-        'noplaylist': True,
-        'format': 'bestaudio/best',
-        'keepvideo': False,
-        'outtmpl': output_path,
-        'postprocessors': [{
-            'key': 'FFmpegExtractAudio',
-            'preferredcodec': 'mp3',
-            'preferredquality': '192',
-        }]
-    }
-
-    with youtube_dl.YoutubeDL(options) as ydl:
-        ydl.download([video_info['webpage_url']])
-
-    # returns os system eg. 'nt' for windows
+    options_bin = input(
+        "Enter 1 for audio only, 2 for video only or 3 for both: ")
+
+    class Download_Options:
```

Comment:
It might be worth creating a separate client file called `youtube_client.py`. This would provide more organisation and separation, rather than everything being in `run.py`. Something like this:

```
class YouTubeClient():
    def download_video():
        pass
```

Maybe in the YouTubeClient you could also have the options to keep everything container. The `run.py` file would just take user input and call the appropriate methods from the `YouTubeClient` class.

Let me know your thoughts?